batchflow/tests/classification_metrics_test.py

```diff
@@ -0,0 +1,175 @@
+    assert test_shape == metrics_shape
+
+def test_axis_for_multiclass(metrics_name):
+    """Test on axis=None"""
+    y_true, y_pred = np.array([2, 1]), np.array([[0.1, 0.1, 0.8], [0.1, 0.8, 0.1]])
+    with pytest.raises(ValueError):
+        getattr(cm(y_true, y_pred, fmt='proba', axis=None, num_classes=3), metrics_name)()
+
+#accuracy tests
+
+@pytest.mark.parametrize('y_true,y_pred,acc', [
+    (np.array([1, 1, 0, 1]), np.array([0, 0, 1, 0]), 0.0),
+    (np.array([1, 1, 0, 1]), np.array([1, 1, 1, 0]), 0.5),
+    (np.array([1, 1, 0, 1]), np.array([1, 1, 0, 1]), 1.0)
+])
+def test_accuracy_calculation(y_true, y_pred, acc):
+    """Test on correctness of accuracy calculation"""
+    assert acc == cm(y_true, y_pred, fmt='labels', num_classes=2).accuracy()
+
+def test_accuracy_calculation_multiclass():
+    """Test on correctness of accuracy calculation, multiclass"""
+    y_true, y_pred = np.array([2, 1]), np.array([[0.1, 0.1, 0.8], [0.1, 0.8, 0.1]])
+    accuracy = 1.0
+    assert accuracy == cm(y_true, y_pred, fmt='proba', axis=1, num_classes=3).accuracy()
+
+# f1 score tests
+
+@pytest.mark.parametrize('y_true, y_pred, f1_score', [
+    (np.array([1, 1, 0, 1]), np.array([0, 0, 1, 0]), 0.0),
+    (np.array([1, 1, 0, 0]), np.array([0, 1, 1, 0]), 0.5),
+    (np.array([1, 1, 1, 0]), np.array([1, 1, 1, 0]), 1.0)
+])
+def test_f1_calculation(y_true, y_pred, f1_score):
+    """Test on correctness of f1_score calculation"""
+    assert f1_score == cm(y_true, y_pred, fmt='labels', num_classes=2).f1_score()
+
+def test_f1_calculation_multiclass():
+    """Test on correctness of f1 score calculation, multiclass"""
+    y_true, y_pred = np.array([2, 1]), np.array([[0.1, 0.1, 0.8], [0.1, 0.8, 0.1]])
+    f_1 = 1.0
+    assert f_1 == cm(y_true, y_pred, fmt='proba', axis=1, num_classes=3).f1_score()
+
+# true positive rate tests
+
+@pytest.mark.parametrize('y_true, y_pred, tpr', [
+    (np.array([1, 1, 0, 1]), np.array([0, 0, 1, 0]), 0.0),
+    (np.array([1, 1, 0, 0]), np.array([0, 1, 1, 0]), 0.5),
+    (np.array([1, 1, 0, 0]), np.array([1, 1, 1, 0]), 1.0)
+])
+def test_tpr_calculation(y_true, y_pred, tpr):
+    """Test on correctness of true positive rate calculation"""
+    assert tpr == cm(y_true, y_pred, fmt='labels', num_classes=2).true_positive_rate()
+
+def test_tpr_calculation_multiclass():
+    """Test on correctness of true positive rate calculation, multiclass"""
+    y_true, y_pred = np.array([2, 1]), np.array([[0.1, 0.1, 0.8], [0.1, 0.8, 0.1]])
+    tpr = 1.0
+    assert tpr == cm(y_true, y_pred, fmt='proba', axis=1, num_classes=3).true_positive_rate()
+
+# false positive rate tests
+
+@pytest.mark.parametrize('y_true, y_pred, fpr', [
+    (np.array([0, 0, 0, 1]), np.array([0, 0, 0, 0]), 0.0),
+    (np.array([1, 1, 0, 0]), np.array([0, 1, 1, 0]), 0.5),
+    (np.array([0, 0, 0, 1]), np.array([1, 1, 1, 1]), 1.0)
+])
+def test_fpr_calculation(y_true, y_pred, fpr):
```

Comment:
`test_%metricname%_calculation` and `test_%metricname%_calculation_multiclass` are identical for all metrics.
Use parametrization over metric names to group them into two test.